src/modals/_SearchModal.jsx

```diff
@@ -146,18 +140,21 @@ const Search = ({ fetchSearchItems, onItemSelected }) => {
                 key={result.item.path}
                 isSelectable
                 metaData={result}
-                onSelect={(event, metaData) => handleSelect(metaData, onItemSelected, disclosureManager)}
+                onSelect={(event, metaData) => {
+                  onRequestClose();
+                  history.push(metaData.item.path);
+                }}
               >
                 {searchItem(result)}
               </Item>
             ))
           }
         </InfiniteList>
       )}
-    </ContentContainer>
+    </ApplicationModal>
   );
 };
 
-Search.propTypes = propTypes;
+SearchModal.propTypes = propTypes;
 
-export default Search;
+export default SearchModal;
```

Comment:
Should the search.module.scss theme files be renamed to SearchModal.module.scss?

Reply:
Yes it should